Spigot-Server-Patches/0543-Always-track-EnderDragon.patch

```diff
@@ -0,0 +1,34 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: kickash32 <kickash32@gmail.com>
+Date: Fri, 12 Jun 2020 21:18:33 -0400
+Subject: [PATCH] Always track EnderDragon
+
+
+diff --git a/src/main/java/org/spigotmc/TrackingRange.java b/src/main/java/org/spigotmc/TrackingRange.java
+index 43b5ed8e396e5312f7de1f160f596f58baead28a..eed715c446a3febd660c157029f7445900bbee83 100644
+--- a/src/main/java/org/spigotmc/TrackingRange.java
++++ b/src/main/java/org/spigotmc/TrackingRange.java
+@@ -1,5 +1,6 @@
+ package org.spigotmc;
+ 
++import com.destroystokyo.paper.PaperWorldConfig; // Paper
+ import net.minecraft.server.Entity;
+ import net.minecraft.server.EntityEnderDragon; // Paper
+ import net.minecraft.server.EntityExperienceOrb;
+@@ -23,6 +24,7 @@ public class TrackingRange
+     public static int getEntityTrackingRange(Entity entity, int defaultRange)
+     {
+         SpigotWorldConfig config = entity.world.spigotConfig;
++        PaperWorldConfig paperConfig = entity.world.paperConfig; // Paper
+         if ( entity instanceof EntityPlayer )
+         {
+             return config.playerTrackingRange;
+@@ -45,7 +47,7 @@ public class TrackingRange
+             return config.miscTrackingRange;
+         } else
+         {
+-            if (entity instanceof EntityEnderDragon) return defaultRange; // Paper - enderdragon is exempt
++            if (entity instanceof EntityEnderDragon) return Math.max(Math.max(config.viewDistance * 16, defaultRange),  paperConfig.noTickViewDistance); // Paper - enderdragon is exempt
```

Comment:
can't use config values as these are variable on runtime

Reply:
This took me too long to figure out what you were talking about x.x